### PR TITLE
Release v0.40.0

### PR DESCRIPTION
**Pull request by khanhtc1202**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: docs/content/en/blog/releases/v0.40.0.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+## Changes since v0.39.0
+### Notable Changes
+
+* Enable to restart Piped from the PipeCD console [#3988](https://github.com/pipe-cd/pipecd/pull/3988)
+* Allow config MySQL database via config URL ([#4017](https://github.com/pipe-cd/pipecd/pull/4017))
+* Fix K8s resources are not pruned even if running Quick Sync ([#3991](https://github.com/pipe-cd/pipecd/pull/3991))
+
+### Internal Changes
+
+* Release v0.40.0
```
Comment (knanao) [inline docs/content/en/blog/releases/v0.40.0.md:18]:
```suggestion
```

**Review by nghialv — APPROVED** [review]
🚀

File: docs/content/en/blog/releases/v0.40.0.md (hunk 1 above)
Comment (knanao) [reply docs/content/en/blog/releases/v0.40.0.md:18]: oh, it's too late.

Comment (khanhtc1202) [reply docs/content/en/blog/releases/v0.40.0.md:18]: We can remove it later with docs stub fix 👍

**Pull request merged**